Triangulation_2/examples/Triangulation_2/triangulation_prog1.cpp

```diff
@@ -23,5 +23,4 @@ int main() {
     do { std::cout << vc->point() << std::endl;
     }while(++vc != done);
   }
-  return 0;
-}
+  return 0;
```

Comment:
this file should not be changed

Reply:
@sloriot I have just checkout the file from earlier commits so that it is removed from changed files, I am sorry that I missed this before.